addons/mail/static/src/components/discuss_sidebar_mailbox/discuss_sidebar_mailbox.xml

```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="mail.DiscussSidebarMailbox" owl="1">
-        <t t-if="mailbox">
+        <t t-if="discussSidebarMailboxView">
```

Comment:
unrelated to diff I guess but guard could be removed

Reply:
Changes done, but it's gonna be a bit annoying for anyone to rebase in case it made some changes in this file.

Reply:
Indeed, it's gonna be annoying to matter what, might as well do it now that half the file is changed anyway if we have to do grouped PR.